chore(kt-devnet):  improve memstore devnet justfile and yaml

samlaf opened the pull request:
Add new justfile commands to get status, cause proxy to failover, restart batcher.
These will be needed to write tests.
Also fixed op-geth's version b/c there is a bug in latest version.

samlaf committed:
chore(memstore-devnet): add justfile commands + fix op-geth version (bug in latest version)

bxue-l2 commented on kurtosis-devnet/eigenda-memstore.yaml:
```diff
@@ -24,7 +24,9 @@ optimism_package:
-        el_image: ""
+        # latest tag is currently broken until the next stable release, see https://github.com/ethereum-optimism/op-geth/pull/515
+        # Also see discussion in https://discord.com/channels/1244729134312198194/1260624141497798706/1342556343495692320
+        el_image: "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-geth:optimism"
```
is "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-geth:optimism" latest?

I have tried "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-geth:v1.101411.8" worked. Should we use it as default? then update later

samlaf replied:
It's not. protolambda suggested I use the optimism tag. I've been using it and it works. Not exactly sure what it is, think its a moving target like latest but is not affected by their latest tagging bug.

samlaf committed:
chore(kt-devnet): rename and refactor some justfile commands

samlaf committed:
ci(kt-devnet): fix ci

1. stop running holesky devnet given holesky is borked
2. update just commands to use "-start" suffix
